lib/perl/Genome/Qc/Tool/BamFile.pm

```diff
@@ -18,7 +14,11 @@ sub supports_streaming {
 
 sub cmd_line {
     my $self = shift;
-    return (qw(samtools view), $self->input_file);
+    return (qw(samtools view), $self->gmt_params->{input_file});
```

Comment:
I talked to @apregier about this suggestion and we believe that samtools is stable enough that we are safe to call samtools directly. Moreover, `G::M::T::Sam::BamToSam` also just calls `samtools view` directly and ignores the `user_version` parameter.